crypto/asn1/a_gentm.c

```diff
@@ -154,13 +154,10 @@ int ASN1_GENERALIZEDTIME_print(BIO *bp, const ASN1_GENERALIZEDTIME *tm)
         }
     }
 
-    if (BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
-                   _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
-                   (gmt) ? " GMT" : "") <= 0)
-        return (0);
-    else
-        return (1);
- err:
+    return BIO_printf(bp, "%s %2d %02d:%02d:%02d%.*s %d%s",
+                      _asn1_mon[M - 1], d, h, m, s, f_len, f, y,
+                      (gmt) ? " GMT" : "") > 0;
+err:
```

Comment:
Should make a proposal to fix this to something like "labels start with no space..."?